apis_configs/fence_settings.py

```diff
@@ -113,19 +114,6 @@ def get_from_dict(dictionary, key, default=''):
     HTTP_PROXY = data['HTTP_PROXY']
     dbGaP = data.get('dbGaP',DEFAULT_DBGAP)
 
-    CIRRUS_CFG = {}
```

Comment:
We still need this configuration if it exists... this will break DCF

Reply:
You can remove the print statement from get_from_dict if you don't want the warnings

Reply:
I moved it to line 104 @Avantol13

Reply:
@thanh-nguyen-dang The entire block of code under this line cannot be deleted. We need to read these configuration variables from the `fence_credentials.json` file. These are all required for fence to operate correctly for Google-related endpoints. This will break existing functionality. If you want to remove the warnings, just remove the print statement in `get_from_dict`